README.md

```diff
@@ -1,53 +1,45 @@
-`odo` - Developer-focused CLI for Kubernetes and OpenShift
+`odo` - Fast iterative application development deployed immediately to your Kubernetes cluster
```

Comment:
I just noticed that this same statement is also there on [odo.dev](https://odo.dev) home page. It doesn't seem right to me. There's no way for someone to know from the landing page that odo is a CLI tool. I would suggest something like:
```suggestion
`odo` - CLI tool for fast & iterative application development; deployed immediately to your Kubernetes cluster
```

Reply:
But it's also too long to put on the main banner on the landing page.

Reply:
We don't say CLI tool because it's also a plugin / API